sky/provision/runpod/api/commands.py

```diff
@@ -0,0 +1,117 @@
+"""This module provides functions to generate GraphQL mutations for deploying
```

Comment:
Could we have a link to the original reference, e.g. the runpod python API

Reply:
added